index.html

```
@@ -7,126 +7,7 @@
-        #blocklyDiv {
-            flex: 1;
-            width: 100%;
-            height: 100%;
-            background-image: 
-                linear-gradient(0deg, transparent 24%, rgba(200, 200, 200, 0.3) 25%, rgba(200, 200, 200, 0.3) 26%, transparent 27%, transparent 74%, rgba(200, 200, 200, 0.3) 75%, rgba(200, 200, 200, 0.3) 76%, transparent 77%, transparent),
-                linear-gradient(90deg, transparent 24%, rgba(200, 200, 200, 0.3) 25%, rgba(200, 200, 200, 0.3) 26%, transparent 27%, transparent 74%, rgba(200, 200, 200, 0.3) 75%, rgba(200, 200, 200, 0.3) 76%, transparent 77%, transparent);
-            background-size: 40px 40px;
-            background-position: 0 0;
-        }
-    </style>
+    <link rel="stylesheet" href="./style.css"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)